docs/learn/learn-xcvm.md

```diff
@@ -187,6 +187,19 @@ Keep in mind that this location is specific to this interaction. The identities
 if this location was defined on another consensus system, such as Kusama. On other consensus
 systems, such as Ethereum, it won't be able to interpret it.
 
+### UniversalLocation in XCM
+
+While a `MultiLocation` may handle relative paths in _local_ consensus, a `UniversalLocation` refers
+to any global consensus system. A global consensus system is an entity like another relay chain,
```

Comment:
"...is an entity which provides its own top-level consensus through some non-derivative consensus algorithm able to exist without reference to any other singleton data system".